NitroxClient/Helpers/NitroxProtobufSerializer.cs

```diff
@@ -13,13 +14,15 @@ public class NitroxProtobufSerializer
         public readonly Dictionary<Type, int> NitroxTypes = new Dictionary<Type, int>();
 
         private readonly Dictionary<Type, int> knownTypes;
+        private readonly HashSet<string> componentWhitelist;
 
         protected RuntimeTypeModel Model => model;
 
         public NitroxProtobufSerializer(params string[] assemblies)
         {
             model = TypeModel.Create();
             knownTypes = (Dictionary<Type, int>)typeof(ProtobufSerializerPrecompiled).GetField("knownTypes", BindingFlags.NonPublic | BindingFlags.Static).GetValue(null);
+            componentWhitelist = (HashSet<string>)typeof(ProtobufSerializer).GetField("componentWhitelist", BindingFlags.NonPublic | BindingFlags.Static).GetValue(null);
```

Comment:
Maybe use our `Reflect`-Helper here?

Reply:
Not even needed to use `Reflection` here. The fields are directly accessible because of publicized assemblies.